tests/BladeTest.php

```diff
@@ -28,12 +28,12 @@ public function all_blade_directives_will_evaluate_false_when_there_is_nobody_lo
         $roles = [$role];
         $elserole = 'na';
 
-        $this->assertEquals('does not have permission', $this->renderView('can', ['permission' => $permission]));
+        $this->assertEquals('does not have permission', $this->renderView('can', compact('permission')));
```

Comment:
I 'get' why you replaced this line with `compact()`, but perhaps it can help add clarity if we keep the original line AND add `compact()`. I know sometimes people read tests for examples, and showing both syntaxes would confirm that both can be used.
Or maybe I'm being pedantic?  :)

Reply:
Both can be used, but  I change it for consistence, I'm going to revert the change